pennylane/devices/default_qubit.py

```diff
@@ -733,7 +810,9 @@ def _apply_unitary_einsum(self, state, mat, wires):
         )
 
         # We now put together the indices in the notation numpy's einsum requires
-        einsum_indices = f"{new_indices}{affected_indices},{state_indices}->{new_state_indices}"
+        einsum_indices = (
+            f"...{new_indices}{affected_indices},...{state_indices}->...{new_state_indices}"
```

Comment:
This syntax allows for
1. The matrix being broadcasting `"z{new_indices}{affected_indices},{state_indices}->z{new_state_indices}"`
2. The state being broadcasting `"{new_indices}{affected_indices},z{state_indices}->z{new_state_indices}"`
3. Both being broadcasting `"z{new_indices}{affected_indices},z{state_indices}->z{new_state_indices}"`